applications/conceptual-model-editor/src/action/actions-react-binding.tsx

```diff
@@ -222,13 +227,20 @@ const noOperationActionsContext = {
   //
   addEntitiesFromSemanticModelToVisualModel: noOperation,
   addClassNeighborhoodToVisualModel: noOperation,
+  layoutActiveVisualModel: noOperationAsync,
   diagram: null,
 };
 
 function noOperation() {
   LOG.error("[ACTIONS] Using uninitialized actions context!");
 }
 
+// TODO PRQuestion: I added back the async operation for layout action (but maybe it isn't action?).
+function noOperationAsync() {
+  LOG.error("[ACTIONS] Using uninitialized actions context!");
+  return Promise.resolve();
+}
+
```

Comment:
Wasn't sure why it was removed in the first place + the question about layouting being an action